2dfft_utils/misc/fits_to_text.py

```diff
@@ -0,0 +1,25 @@
+'''
+    fitstotext.py
+    Purpose: Converts .fit files to .txt, replacing IRAF cl script.
+    Input: FITS image files. The string in line 16 can be modified to reflect your needs and file
+    naming conventions.
+    Output: .txt files with the pixel values for the corresponding image and the same basename.
+    Author: Erik Monson
+    
+    Last modified May 29, 2015
+'''
+
+import glob
+from pyraf import iraf
+
+#Get filenames of all the suitable fits files in working directory
+fits_list_all = glob.glob("*crop.fit")
```

Comment:
How about we just make this `"*fit"`?  Better yet, check for any `*fits` as well (since some people will have FITS files with an `s` in the extension).